apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -159,13 +161,12 @@ const NftViewer = ({ route }: Props) => {
     metadata?: NFTMetadataResponse["result"] & NFTCollectionMetadataResponse["result"];
   };
   const { t } = useTranslation();
-  const navigation =
-    useNavigation<
-      CompositeNavigationProp<
-        StackNavigatorNavigation<NftNavigatorParamList, ScreenName.NftViewer>,
-        StackNavigatorNavigation<BaseNavigatorStackParamList>
-      >
-    >();
+  const navigation = useNavigation<
```

Comment:
Nothing changed here, prettier was not running in this file so it reformatted other things when I got it to run.